### ISSUE TITLE
Feed sorting/filtering for unauthenticated users

### ISSUE THREAD

**Issue by Naraujo13** [FightPandemics/FightPandemics#376]
As an unauthenticated user, I can access the feed and get posts sorted by location based on my IP address.

- Get location from IP

- Change `/posts` to sort by generated location 

- Change `/posts` to filter posts based on visibility (city, state, country and worldwide) with the generated location

**Comment by joshmorel** [comment]
Regarding the comment in #1303 about sorting/filtering by locaiton for non-logged in user @rabbi08 cc @ppagarawal it is tagged as being post-MVP so basically not expected to be worked on anytime soon. However, I'm not sure if that was correct.

**Comment by ppagarawal** [comment]
@joshmorel the product team is still working on prioritization of different features at the moment. There are more important features which need to be built and you can focus on them if this will be time consuming

**Comment by joshmorel** [comment]
@ppagarawal Makes sense. I don't plan on tackling this one. But someone else can work on this one - not sure @Naraujo13 if you were still planning to do so.

**Comment by Naraujo13** [comment]
If someone has the time to work on this, feel free, I'm a bit short currently
